app/src/main/java/com/android/unio/ui/authentication/Welcome.kt

```diff
@@ -143,7 +144,32 @@ fun WelcomeScreen(userViewModel: UserViewModel) {
                       else PasswordVisualTransformation(),
               )
 
-              Spacer(modifier = Modifier.size(70.dp))
+              TextButton(
+                  onClick = {
+                    authViewModel.sendEmailResetPassword(
+                        email,
```

Comment:
Watch out ! I tested your implementation and if email is null (the user hasn't written anything) the app will crash ! 
You should check the value of the email variable before sending any reset password, typically this can be done with the `isValidEmail` function located in `Auth.kt`!

Reply:
Yes, you are right, my bad